engine/models_test.go

```diff
@@ -30,7 +30,7 @@ func TestModelName_String(t *testing.T) {
 		assert.Equal(t, "utxo", ModelUtxo.String())
 		assert.Equal(t, "xpub", ModelXPub.String())
 		assert.Equal(t, "contact", ModelContact.String())
-		assert.Len(t, AllModelNames, 10)
+		assert.Len(t, AllModelNames, 11)
```

Comment:
I don't know why we have this test, but maybe you should add an assertion for webhook model?